### PR TITLE
Fix bunch of slipped E2E_NAME flags

### PR DESCRIPTION
**Pull request by krzyzacy**
Try to catch them in the unit test.

Cover https://github.com/kubernetes/test-infra/pull/1833 and more other jobs has this issue.

### CODE REVIEW

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -1599,16 +1599,26 @@ def testEnvsNoExport(self):
...
-                # FOO=aaa"bbb"aaa -> good
-                # FOO=a # BAR -> bad (no inline comments in env files)
-                m = re.match(r'[0-9A-Z_]+=([^\"]$|[^\n#]+[^\"\n#]$)', line)
-                empty = (line.strip() == '')
-                comment = line.startswith('#')
-                if not (m or empty or comment):
-                    self.fail('Job %s contains invalid env: %s' % (job, line))
+                line = line.strip()
+                self.assertFalse(line.endswith('\\'))
+                if line == '':
```
Comment (rmmh) [inline jenkins/bootstrap_test.py]: 'if not line'

File: jenkins/bootstrap_test.py [hunk 2]
```diff
@@ -1599,16 +1599,26 @@ def testEnvsNoExport(self):
...
+                    continue
+                if line.startswith('#'):
+                    continue
+                if not re.match(r'[0-9A-Z_]+=[^\n]+', line):
+                    self.fail('[%s]: Env %s: need to follow FOO=BAR pattern' % (job, line))
```
Comment (rmmh) [inline jenkins/bootstrap_test.py]: throughout: consider using %r for printing line to escape quotes

File: jenkins/bootstrap_test.py (hunk 1 above)
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: done.

File: jenkins/bootstrap_test.py (hunk 2 above)
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: done.

**Comment by krzyzacy** [comment]
@wojtek-t this should fix the `gke-large` and `gce-enormous` flags, I guess I'll give you a chance to take another look and feel free to merge it if it's LG to you.

File: jobs/config.json [hunk 3]
```diff
@@ -1685,23 +1698,29 @@
...
 
 "ci-kubernetes-e2e-gke-large-teardown": {
   "scenario": "kubernetes_e2e",
   "args": [
     "--env-file=platforms/gke.env",
-    "--env-file=jobs/ci-kubernetes-e2e-gke-large-teardown.env"
+    "--env-file=jobs/ci-kubernetes-e2e-gke-large-teardown.env",
+    "--cluster=gke-large-deploy",
+    "--up=false",
+    "--test=false"
```
Comment (wojtek-t) [inline jobs/config.json]: From scalability-related jobs it LGTM.
However, you modified 16 ".env" files, but only 13 jobs here, so something is probably missing...

Comment (krzyzacy) [inline jobs/config.json]: good catch!

Comment (krzyzacy) [inline jobs/config.json]: done

**Pull request merged**